### PR TITLE
Add differential 1..4-channel ADCs

### PR DESCRIPTION
**Pull request by dbrgn**

### CODE REVIEW

File: cmp/31e1fdb4-8802-4e2e-881a-4965aaa84394/component.lp [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+   (symbol 6af97d05-bdca-4efa-b567-aa6129895425)
+   (position 10.16 0.0) (rotation 0.0) (required true) (suffix "ADC")
+   (pin 0e0dd2a1-d269-441a-b4df-73a7affb0162 (signal 9ec4d519-1c2a-42c9-9d16-355189f3ad22) (text signal))
+   (pin 47720781-c4de-417b-87a6-1223da2c736f (signal 933160f8-1590-448f-b206-ba39421515dd) (text signal))
+   (pin b2fac1ac-6a19-40a5-ab07-5c5de1759293 (signal 2480ef1d-260c-4926-a233-50508adb34a9) (text signal))
+   (pin e4102d93-eb84-4b8e-90b2-d935d6b63c6d (signal 3e11bf60-08df-4a58-ab5e-fafa3892c01a) (text signal))
+  )
+  (gate a744ed58-177f-45cc-af96-12ddb79b2729
+   (symbol 6d6226bc-0498-4077-999d-716896a7bd4e)
+   (position -10.16 10.16) (rotation 0.0) (required true) (suffix "CH1")
```
Comment (ubruhin) [inline cmp/31e1fdb4-8802-4e2e-881a-4965aaa84394/component.lp]: It's not required to use all channels:

```suggestion
   (position -10.16 10.16) (rotation 0.0) (required false) (suffix "CH1")
```

File: cmp/31e1fdb4-8802-4e2e-881a-4965aaa84394/component.lp [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+   (pin 92f6bdbf-874b-48c7-91eb-50bbef5185ff (signal 52c6f2c2-89be-4129-b0c9-8558511b492b) (text signal))
+   (pin ea4b33e6-7c57-4410-b3b4-03d3fadfd77e (signal 04f15665-0053-48e6-b6ad-375f48f89c82) (text signal))
+  )
+  (gate 5402fb42-8308-471a-901b-ef2f65e8bcb5
+   (symbol 6d6226bc-0498-4077-999d-716896a7bd4e)
+   (position -10.16 -10.16) (rotation 0.0) (required true) (suffix "CH2")
```
Comment (ubruhin) [inline cmp/31e1fdb4-8802-4e2e-881a-4965aaa84394/component.lp]:
```suggestion
   (position -10.16 -10.16) (rotation 0.0) (required false) (suffix "CH2")
```

File: cmp/41541776-0222-437f-be0d-6e6202c7b9bc/component.lp [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+   (symbol 6af97d05-bdca-4efa-b567-aa6129895425)
+   (position 10.16 0.0) (rotation 0.0) (required true) (suffix "ADC")
+   (pin 0e0dd2a1-d269-441a-b4df-73a7affb0162 (signal 2e0aa692-facf-43f9-a442-a207f81ef858) (text signal))
+   (pin 47720781-c4de-417b-87a6-1223da2c736f (signal 4c899dcc-520f-4164-903a-f81a85ff2d85) (text signal))
+   (pin b2fac1ac-6a19-40a5-ab07-5c5de1759293 (signal cb165b3f-0f99-4461-9c13-06e3421aad61) (text signal))
+   (pin e4102d93-eb84-4b8e-90b2-d935d6b63c6d (signal d3f34900-e323-4bba-901a-52bf8419e173) (text signal))
+  )
+  (gate 47d4085d-0fbc-4e1b-85be-11d834ddef95
+   (symbol 6d6226bc-0498-4077-999d-716896a7bd4e)
+   (position -10.16 30.48) (rotation 0.0) (required true) (suffix "CH1")
```
Comment (ubruhin) [inline cmp/41541776-0222-437f-be0d-6e6202c7b9bc/component.lp]:
```suggestion
   (position -10.16 30.48) (rotation 0.0) (required false) (suffix "CH1")
```

File: cmp/41541776-0222-437f-be0d-6e6202c7b9bc/component.lp [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+   (pin 92f6bdbf-874b-48c7-91eb-50bbef5185ff (signal f1979b46-2c7a-4414-b9be-dc53c9832444) (text signal))
+   (pin ea4b33e6-7c57-4410-b3b4-03d3fadfd77e (signal 888c75d3-d2bc-4d5b-9173-51cd54f75272) (text signal))
+  )
+  (gate 13061a62-fee3-4de5-b46a-61a76f591f02
+   (symbol 6d6226bc-0498-4077-999d-716896a7bd4e)
+   (position -10.16 10.16) (rotation 0.0) (required true) (suffix "CH2")
```
Comment (ubruhin) [inline cmp/41541776-0222-437f-be0d-6e6202c7b9bc/component.lp]:
```suggestion
   (position -10.16 10.16) (rotation 0.0) (required false) (suffix "CH2")
```

File: cmp/41541776-0222-437f-be0d-6e6202c7b9bc/component.lp [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+   (pin 92f6bdbf-874b-48c7-91eb-50bbef5185ff (signal 4ae0c14d-be1f-4095-98d5-36d36eb1a6f9) (text signal))
+   (pin ea4b33e6-7c57-4410-b3b4-03d3fadfd77e (signal ace8c80f-f20c-476d-8dc7-bca8468809f7) (text signal))
+  )
+  (gate 395f32b4-3160-4503-9f83-773c431fa7af
+   (symbol 6d6226bc-0498-4077-999d-716896a7bd4e)
+   (position -10.16 -10.16) (rotation 0.0) (required true) (suffix "CH3")
```
Comment (ubruhin) [inline cmp/41541776-0222-437f-be0d-6e6202c7b9bc/component.lp]:
```suggestion
   (position -10.16 -10.16) (rotation 0.0) (required false) (suffix "CH3")
```

File: cmp/e1439ad4-5770-4398-b251-dc12843a328f/component.lp [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+   (symbol 6af97d05-bdca-4efa-b567-aa6129895425)
+   (position 10.16 0.0) (rotation 0.0) (required true) (suffix "ADC")
+   (pin 0e0dd2a1-d269-441a-b4df-73a7affb0162 (signal 5f209d07-2eaa-43b6-85fa-319a815a45e4) (text signal))
+   (pin 47720781-c4de-417b-87a6-1223da2c736f (signal 04de5e85-ac1f-4db7-854d-a9ec91e6b218) (text signal))
+   (pin b2fac1ac-6a19-40a5-ab07-5c5de1759293 (signal a0361753-bb41-45cc-a7ef-bdee609d7987) (text signal))
+   (pin e4102d93-eb84-4b8e-90b2-d935d6b63c6d (signal 8d252cec-bf45-4e7a-9c26-3dc801d7e054) (text signal))
+  )
+  (gate 982536e9-b246-4c77-a498-03a2bdc9202c
+   (symbol 6d6226bc-0498-4077-999d-716896a7bd4e)
+   (position -10.16 17.78) (rotation 0.0) (required true) (suffix "CH1")
```
Comment (ubruhin) [inline cmp/e1439ad4-5770-4398-b251-dc12843a328f/component.lp]:
```suggestion
   (position -10.16 17.78) (rotation 0.0) (required false) (suffix "CH1")
```

File: cmp/e1439ad4-5770-4398-b251-dc12843a328f/component.lp [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+   (pin 92f6bdbf-874b-48c7-91eb-50bbef5185ff (signal dbc2733f-1b54-49dd-8657-3dd8eefbf389) (text signal))
+   (pin ea4b33e6-7c57-4410-b3b4-03d3fadfd77e (signal 652bba1d-2611-4023-96e7-3c793e212310) (text signal))
+  )
+  (gate b6052491-e668-4296-9619-c43fdb6a6cbe
+   (symbol 6d6226bc-0498-4077-999d-716896a7bd4e)
+   (position -10.16 0.0) (rotation 0.0) (required true) (suffix "CH2")
```
Comment (ubruhin) [inline cmp/e1439ad4-5770-4398-b251-dc12843a328f/component.lp]:
```suggestion
   (position -10.16 0.0) (rotation 0.0) (required false) (suffix "CH2")
```

File: cmp/e1439ad4-5770-4398-b251-dc12843a328f/component.lp [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+   (pin 92f6bdbf-874b-48c7-91eb-50bbef5185ff (signal 8d7216b0-fa97-40f2-b15b-e96bec9d97fb) (text signal))
+   (pin ea4b33e6-7c57-4410-b3b4-03d3fadfd77e (signal 85dfbde4-6b1c-4194-887d-a04182b56247) (text signal))
+  )
+  (gate 1fb78cb0-6111-4779-9b5f-7ab196d463ee
+   (symbol 6d6226bc-0498-4077-999d-716896a7bd4e)
+   (position -10.16 -17.78) (rotation 0.0) (required true) (suffix "CH3")
```
Comment (ubruhin) [inline cmp/e1439ad4-5770-4398-b251-dc12843a328f/component.lp]:
```suggestion
   (position -10.16 -17.78) (rotation 0.0) (required false) (suffix "CH3")
```

File: cmp/41541776-0222-437f-be0d-6e6202c7b9bc/component.lp [hunk 9]
```diff
@@ -0,0 +1,85 @@
...
+   (pin 92f6bdbf-874b-48c7-91eb-50bbef5185ff (signal b0b21973-bc00-42bc-9e04-6028b9bb0ef1) (text signal))
+   (pin ea4b33e6-7c57-4410-b3b4-03d3fadfd77e (signal e9d713e2-f216-4f2e-9ce6-efe2b863e4c0) (text signal))
+  )
+  (gate 9d7e88e4-cf55-43d1-b80e-2189287b41c8
+   (symbol 6d6226bc-0498-4077-999d-716896a7bd4e)
+   (position -10.16 -30.48) (rotation 0.0) (required true) (suffix "CH4")
```
Comment (ubruhin) [inline cmp/41541776-0222-437f-be0d-6e6202c7b9bc/component.lp]:
```suggestion
   (position -10.16 -30.48) (rotation 0.0) (required false) (suffix "CH4")
```

**Comment by dbrgn** [comment]
@ubruhin fixed and rebased!

**Comment by ubruhin** [comment]
Oops, I just realized that the symbol name/description/keywords are actually not correct since the symbol has 6 pins, not 5 :sob:

Otherwise everything looks good :+1:

**Comment by dbrgn** [comment]
Oops, right :smile:

**Comment by dbrgn** [comment]
@ubruhin fixed!

**Pull request merged**